SECURITY_RELEASE_PROCESS.md

```diff
@@ -164,6 +164,11 @@ issue fixed for your respective distribution's users.
 Before any information from the list is shared with respective members of your team required to fix
 said issue, they must agree to the same terms and only find out information on a need-to-know basis.
 
+The embargo applies to information shared, source code and binary images. It is a violation of the
```

Comment:
Can you bold the sentence starting with "It is a violation..." I want to make sure this super clear as I'm not sure folks generally think about this.